src/test_driver/openiotsdk/integration-tests/lock-app/test_app.py

```diff
@@ -67,7 +80,7 @@ def test_commissioning(device, controller):
     assert disconnect_device(devCtrl, nodeId)
 
 
-LOCK_CTRL_TEST_PIN_CODE = 12345
+LOCK_CTRL_TEST_PIN_CODE = b"123456"
```

Comment:
Why do we change the test pin code of the lock app here ?

Reply:
We need to change it because it was a bug before. Now we pass it as a binary array (6 bytes). Before it was a string which means 5 characters + end line sign (also 6 bytes). The required pin length is minimum 6 bytes.